template/template.go

```diff
@@ -15,16 +15,15 @@ package template
 
 import (
 	"bytes"
+	tmplhtml "html/template"
```

Comment:
this was formatted like this by `goimports`? weird

Reply:
It was most probably a manual renaming.